src/components/organims/welcomeByOrg.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import {Redirect, navigate} from '@reach/router';
+import AppMinimal from '../layout/AppMinimal';
+import Seo from '../molecules/Seo';
+import Button from '../atoms/Button';
+import styles from './welcome.module.css';
+import gql from 'graphql-tag';
+
+const getOrgInfo = gql`
+  query($slug: String!) {
+    organizationWithSlug(slug: $slug) {
+      slug,
+      name,
+      pictureUrl,
+    }
+  }
+`;
+
+export class WelcomeByOrg extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      title: 'Your organization name',
+      description: 'Your organization description',
+      details: 'Your organization details',
+      email: 'name.org',
+      pictureUrl: '',
+      hasDownloaded: false,
+    }
+  }
+  componentWillMount(){
+    const { data, organizationSlug } = this.props;
+    const parsedUrl = new URL(window.location.href);
+    // If url contain ?dl=true just dl the file too !
+    if(parsedUrl && parsedUrl.searchParams.get('dl')){
+      window.location= data.button_url.url;
+    }
+  }
+  componentDidMount(){
+    const { organizationSlug, client, data } = this.props;
+    client
+    .query({
+      query: getOrgInfo,
+      variables: { slug: organizationSlug}
+    })
+    .then(res=>{
+      const {name, pictureUrl, slug} = res.data.organizationWithSlug;
+      if(!name || !slug || ! pictureUrl) {
```

Comment:
Ah yes can use station logo when no other one is provided . 
If organization has no name should I use it's domain to implicitly get its name (the *name* is use)? It can only work if domain sticky contain the name.

Reply:
> Ah yes can use station logo when no other one is provided.

Not sure it was clear: I meant using **only** Station logo (and thus center it and removes "&").

> If organization has no name should I use it's domain to implicitly get its name (the name is use)? It can only work if domain sticky contain the name.
I don't get it sorry.
I meant when `name` is null, use `domain`.
Implicitely: if `domain` is not available (should not happen in theory), the query will fail and you should display a 404.